appveyor.yml

```diff
@@ -0,0 +1,68 @@
+version: 3.0.5.{build}
+skip_tags: true
+shallow_clone: true
+cache:
+  - C:\ProgramData\chocolatey\bin                                                                                                     
+  - C:\ProgramData\chocolatey\lib
+  - C:\Users\appveyor\.ccm\repository
+  - C:\Users\appveyor\ccm
+os: Visual Studio 2015
+platform:
+  - Any CPU
+configuration:
+  - Debug
+environment:
+  NUNIT_PATH: nunit-console
+  matrix:
+    - CASSANDRA_VERSION: 3.4
+
+    - CASSANDRA_VERSION: 3.0.4
+
+    - CASSANDRA_VERSION: 2.2.5
+
+matrix:
+  fast_finish: true   
```

Comment:
For now, we should remove the fail fast, in order to see all the range of possible failures (inconsistent)